nat46/modules/nat46-module.c

```diff
@@ -50,6 +57,10 @@
 #define NAT46_VERSION __DATE__ " " __TIME__
 #endif
 
+#ifdef PROTO_NETLINK
+#define NETLINK_USER 31
```

Comment:
What is the source of the magic value "31" ? Does the code work if you change it "42" here ? If not - it is probably defined somewhere as 31, so that file would need to be included instead of the #define here...

Reply:
Thanks for the feedback.  I'll keep your suggestion in mind as I update my patch.